src/components/ReportActionItem/IOUPreview.js

```diff
@@ -164,8 +160,6 @@ const IOUPreview = (props) => {
     };
 
     const showContextMenu = (event) => {
-        // Use action prop to check if we are in IOUDetailsModal,
-        // if it's true, do nothing when user long press, otherwise show context menu.
         if (!props.action) {
             return;
         }
```

Comment:
I think that we can get rid of this conditional right? We always want to show the context menu now, because the preview is only rendered in the chat view?